### PR TITLE
Added support for Aeria

### PR DESCRIPTION
**Pull request by DarkGL**
https://github.com/aeria-org/aeria

Closes https://github.com/moltar/typescript-runtime-type-benchmarks/pull/1218

### CODE REVIEW

File: cases/aeria/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import type { Collection } from '@aeriajs/types';
+
+declare global {
+  type Collections = Record<string, Collection>;
+}
```
Comment (moltar) [inline cases/aeria/index.d.ts:5]: Could you please add a comment about why this is needed for future selves :)

File: cases/aeria/index.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
+import { silentValidator } from '@aeriajs/validation';
+import { createCase } from '../../benchmarks';
+
+import {} from './index.d';
```
Comment (moltar) [inline cases/aeria/index.ts:4]:
```suggestion
import './index.d';
```

Should be sufficient, no?

File: cases/aeria/index.d.ts (hunk 1 above)
Comment (DarkGL) [reply cases/aeria/index.d.ts]: Actually, it seems that those types are not necessary at all, good catch thank you :)

File: cases/aeria/index.ts (hunk 2 above)
Comment (DarkGL) [reply cases/aeria/index.ts]: https://github.com/moltar/typescript-runtime-type-benchmarks/pull/1347#discussion_r1825191962

**Review by moltar — APPROVED** [review]

**Pull request merged**